Lib/test/test_fstring.py

```diff
@@ -780,5 +780,19 @@ def test_dict(self):
         self.assertEqual(f'{d["foo"]}', 'bar')
         self.assertEqual(f"{d['foo']}", 'bar')
 
+    def test_backslash_char(self):
+        # Check eval of a backslash followed by a control char.
+        # As part of bpo-30529, this used to raise an assert in pydebug mode.
+        for i in range(1, 32):
+            with self.subTest(i=i):
+                # This will result in a backslash followed by the control
```

Comment:
A backslash followed by the control character (except '\r' and '\n') produces a DeprecationWarning.

I would test just a case of `'\\\n'`. Maybe also `'\\\r'` and `'\\\r\n'` if you prefer.